resources/views/privacy.blade.php

```diff
@@ -14,25 +14,25 @@
 <div class="mw8 center ph3 mb6">
   <div class="cf">
     <div class="w-100 w-60-ns center">
-      <p class="measure lh-copy">Monica is an open source project. The hosted version has a premium plan that lets us collect money so we can pay for the servers and additional servers, but the main goal is not to make money (otherwise we wouldn't have opened source it).</p>
+      <p class="measure lh-copy">Monica is an open-source project. The hosted version has a premium plan that lets us collect money so we can pay for the servers and additional services, but the main goal is not to make money (otherwise we wouldn't have open-sourced it).</p>
```

Comment:
I'm not sure about `servers` -> `services`, let me know if this is indeed what you meant

Reply:
Thanks so much @OlegSmelov